hack/jenkins/e2e.sh

```diff
@@ -84,15 +82,13 @@ ${GCE_DEFAULT_SKIP_TEST_REGEX}\
 |NetworkingNew\
 |Nodes\sNetwork\
 |Nodes\sResize\
-|MaxPods\
+|MaxPods.*\
```

Comment:
nit: ".*" is not needed here - all regexps are treated as if they have .* at the beginning and at the end